_code-samples/batch/js/singleAccountBatch.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * Single Account Batch Transaction Example
+ *
+ * This example demonstrates how to use the Batch transactions feature (XLS-56)
+ * to create a single-account batch transaction that sends payments
+ * to multiple destinations in one atomic operation.
+*/
+
+import xrpl from "xrpl"
+import { BatchFlags } from 'xrpl/dist/npm/models/transactions/batch.js'
+import { GlobalFlags } from 'xrpl/dist/npm/models/transactions/common.js'
```

Comment:
FYI, I know this looks weird. Unfortunately, they're not exported yet in the current version of xrpl.js (4.4.2), hence why I imported them this way.

This change will make them available: https://github.com/XRPLF/xrpl.js/pull/3064.
Likely the next release of the SDK though.